tests/neg/t1625.check

```diff
@@ -0,0 +1,8 @@
+-- [E040] Syntax Error: tests/neg/t1625.scala:2:20 ---------------------------------------------------------------------
+2 |  def foo(x: String*, y: String*, c: String*): Int // error: an identifier expected, but ',' found // error: an identifier expected, but ',' found
+  |                    ^
+  |                    an identifier expected, but ',' found
+-- [E040] Syntax Error: tests/neg/t1625.scala:2:32 ---------------------------------------------------------------------
+2 |  def foo(x: String*, y: String*, c: String*): Int // error: an identifier expected, but ',' found // error: an identifier expected, but ',' found
+  |                                ^
```

Comment:
This is an example of better error messages IMHO: you get an error on both invalid occurrences.